src/shared/dropdown-list/dropdown-list.styles.tsx

```diff
@@ -245,11 +277,23 @@ export const ResultStateContainer = styled.div`
     align-items: center;
 `;
 
-export const ResultStateText = styled(Text.Body)``;
+export const ResultStateText = styled.text<ResultStateProps>`
+    ${(props) =>
+        TextStyleHelper.getTextStyle(
+            props.$variant === "small" ? "BodySmall" : "Body",
+            "regular"
+        )}
+`;
 
-export const LabelIcon = styled(ExclamationCircleFillIcon)`
+export const LabelIcon = styled(ExclamationCircleFillIcon)<IconProps>`
+    ${(props) => {
+        // 1rem 16px, 1.5rem 24px
```

Comment:
no need to add the comments on the unit conversion here, as the rem->px is a standard convention in this repo